server/planning/assignments/assignments.py

```diff
@@ -480,6 +480,12 @@ def send_assignment_notification(self, updates, original=None, force=False):
             else None
         )
 
+        coverage_language = assignment.get("planning", {}).get("language")
```

Comment:
Seems like `coverage_language` is only used inside the following `if` statement, might be good to localise it there to indicate it's only used in that statement